code/game/objects/items/clown_items.dm

```diff
@@ -167,12 +167,12 @@
 
 /obj/item/bikehorn/airhorn/Initialize()
 	. = ..()
-	AddComponent(/datum/component/squeak, list('sound/items/airhorn2.ogg'=1), 50)
+	AddComponent(/datum/component/squeak, /datum/outputs/airhorn, 50)
 
 //golden bikehorn
 /obj/item/bikehorn/golden
 	name = "golden bike horn"
-	desc = "Golden? Clearly, it's made with bananium! Honk!"
+	desc = "Golden? Clearly, it's made of bananium! Honk!"

```

Comment:
Why are you changing this?

Reply:
grammar

Reply:
It's correct either way

Reply:
made of is more correct because it implies that's the material the object is made out of rather than an ingredient

Reply:
It is just an ingredient though? And that's a pretty pointless reason to change this regardless.

Reply:
it's just a grammar/formatting change, you don't say "made with gold" for something made of solid gold, you say it's "made of gold"

Reply:
It's a change that doesn't matter either way and is more a matter of taste and has nothing to do with the rest of this pr